ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_mathematical_functions.py

```diff
@@ -148,6 +148,30 @@ def _get_dtype_input_and_vectors(draw):
     return dtype, vec1, vec2
 
 
+# unwrap
+@st.composite
+def _get_dtype_input_vector_axis(draw):
```

Comment:
Helper functions should be in the respective helper modules in tests.

Reply:
@bipinKrishnan - While I was adding the function to helper module, I found an existing function that I can use. So I have used that existing function in the test instead so new helper function isn't required.